src/resources/js/commerce/gateway/paypal/checkout.js

```diff
@@ -350,6 +350,22 @@ tribe.tickets.commerce.gateway.paypal.checkout = {};
 		}
 	};
 
+	/**
+	 * Setup the triggers for Ticket Commerce loader view.
+	 *
+	 * @since TBD
+	 *
+	 * @return {void}
+	 */
+	obj.setupLoader = function() {
+		$document.trigger( tribe.tickets.commerce.customEvents.showLoader );
+
+		// Hide loader when Paypal buttons are added.
+		$document.on( 'DOMNodeInserted', obj.selectors.buttons, function() {
+			$document.trigger( tribe.tickets.commerce.customEvents.hideLoader );
+		} );
```

Comment:
make the function a separate named function on the `obj` and remove the `DOMNodeInserted` event listener after triggering the `hideLoader` event.